cmd/otelcol/config/collector/splunk_config_linux.yaml

```diff
@@ -34,20 +34,17 @@ processors:
   batch:
   # Enabling the memory_limiter is strongly recommended for every pipeline.
   # Configuration is based on the amount of memory allocated to the collector.
-  # The configuration below assumes 2GB of memory. In general, the ballast
-  # should be set to 1/3 of the collector's memory, the limit should be 90% of
-  # the collector's memory up to 2GB, and the spike should be 25% of the
-  # collector's memory up to 2GB. The simplest way to specify the ballast size is
-  # set the value of SPLUNK_BALLAST_SIZE_MIB env variable. This will overrides
+  # In general, the ballast should be set to 1/3 of the collector's memory, the limit
+  # should be 90% of the collector's memory up to 2GB. The simplest way to specify the
+  # ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable. This will override
   # the value of --mem-ballast-size-mib command line flag. If SPLUNK_BALLAST_SIZE_MIB
```

Comment:
Not introduced here or blocking, but don't cli args usually take priority over env vars since they are the most directly applied option for a given process and not potentially inherited?

Reply:
Hmm this comment is wrong -- CLI does take priority. See cmd/otelcol/main.go line 111